go.mod

```diff
@@ -36,10 +36,12 @@ require (
 	github.com/multiformats/go-base32 v0.0.4
 	github.com/multiformats/go-multiaddr v0.4.0
 	github.com/multiformats/go-multihash v0.0.15
+	github.com/regen-network/cosmos-proto v0.3.1 // indirect
 	github.com/spf13/cobra v1.2.1
 	github.com/stretchr/testify v1.7.1-0.20210427113832-6241f9ab9942
 	github.com/tendermint/tendermint v0.34.13
 	go.uber.org/fx v1.14.2
 	go.uber.org/zap v1.19.0
-	google.golang.org/protobuf v1.27.1
 )
+
+replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
```

Comment:
we have to use the protobuf dependency, but we will have to any way since the app depends on it.